Trumpf.Coparoo.Web/Controls/Table/TableDataCell.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Trumpf.Coparoo.Web.Controls.Table
+{
+    using OpenQA.Selenium;
+    using Trumpf.Coparoo.Web;
+
+    /// <summary>
+    /// Table data cell object.
+    /// </summary>
+    public class TableDataCell : ControlObject
```

Comment:
I suggest to rename the class to `TableCell`  and then use it for both cell types (see comment below).